auto_py_to_exe/web/js/i18n.js

```diff
@@ -1977,6 +2044,7 @@ const translationMap = {
         th: 'การดำเนินการนี้จะเขียนทับเอาต์พุตก่อนหน้าในโฟลเดอร์เอาต์พุต\nทำต่อหรือไม่',
         fr: 'Cette action écrasera une sortie précédente dans le dossier de sortie.\nContinuer ?',
         hi: 'यह कार्रवाई आउटपुट फ़ोल्डर में पिछले आउटपुट को अधिलेखित कर देगी। \n क्या आप जारी रखना चाहते हैं?',
+        he: '',
```

Comment:
If there is no translation, it can be left out

Reply:
updated